docs/web/concepts-app-config.mdx

```diff
@@ -0,0 +1,412 @@
+---
+title: App Config
+description: Geocortex Web - Learn about Geocortex Web's configuration concepts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import UseCaseCard from "../../src/components/UseCaseCard";
+import UseCaseContainer from "../../src/components/UseCaseContainer";
+import AppItemSnippet from "../snippets/app-item.mdx";
+import FollowAlongSnippet from "./snippets/follow-along.mdx";
+
+:::note Before learning about app config...
+Check out the [Key Concepts](concepts-key-concepts.mdx).
+:::
+
+While [layouts](concepts-layout.mdx) depict the visual structure of how components are going to appear in the viewer, **app config** is used to define the functional structure within a given component.
+
+-   Think of the app config as the structural materials for building a house (wood, bricks, etc.) and the layout as the blueprints for the house.
+
+-   The materials are the fundamental building blocks that create the house itself, and the blueprints allow for them to be structured together into something that people can interact with.
+
+Every component within the viewer that requires functional, compositional content needs to be configured within the viewer's corresponding app config.
+
+<FollowAlongSnippet />
+
+## Anatomy of an App config.
+
+<AppItemSnippet />
+
+## Linking Layout to App Config
+
+Every component in the layout can be linked to app config with the `config` attribute. The `config` attribute is used to identify the item in the app config that has configuration for the component.
```

Comment:
@eric-parsons you didn't miss it, there is currently nothing on switching between multiple layouts. Personally, I think that's quite an advanced use case for an SDK developer, to have multiple, completely custom layouts they can switch between, and that's why I didn't include it in the scope of this initial pass? It's currently a ticket in the backlog. Totally open to discussing that further though. 

Components sharing the same item type and the same config is very useful. I did include a little note on using the same config for multiple components, (https://deploy-preview-16--geocortex-dev-center.netlify.app/docs/web/concepts-app-config#linking-layout-to-app-config) but it was obviously missed! I'll make that note more important and elaborate on it.

Reply:
I elaborated on using the same config for multiple components and consuming existing model types in under the advanced config section.

Reply:
Thanks for making those changes.

I'm OK with multi-layout being a ticket in the backlog. I disagree that it's advanced... this will be very commonplace. As soon as we have the necessary mechanism for choosing a layout, I suspect many if not most Web templates in App Designer will become multi-layout like Mobile's default is. Also, this will be familiar to anyone coming from GVH. If this is something we are exposing to Clara, then for sure developers are expected to understand it as well. But, it's OK for now if it's missing.